DigitalLearningSolutions.Web/Views/TrackingSystem/CourseSetup/ManageCourse/_CourseSummaryCard.cshtml

```diff
@@ -0,0 +1,29 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+@model CourseSummaryViewModel
+
+<div class="nhsuk-card">
+  <div class="nhsuk-card__content basic-card-content">
+    <dl class="nhsuk-summary-list">
+      <div class="nhsuk-summary-list__row basic-summary-list__row">
+        <dt class="nhsuk-summary-list__key">Current version</dt>

```

Comment:
I'm sure you've run the formatter here, but is there any other reason for these to all be single line rather than the spaced version we're using elsewhere.